docs/new_instance.md

```diff
@@ -1,43 +1,96 @@
 # Adding new scriptworker instances of an existing type
 
-We don't yet have a scriptworker provisioner, so spinning up new instances of a specific type is still a manual process that can definitely use improvement.  Here are docs on how to spin a new instance up.
+We don't yet have a scriptworker provisioner, so spinning up new instances of a specific type is still a manual process that can definitely use improvement.  Here are docs on how to spin a new instance up. These instructions apply to any type of scriptworker that
+already has an instance running, hence configurations already exists.
 
-## signing scriptworker
-### gpg keypair
-If this is a chain of trust enabled scriptworker, you'll need to generate a gpg keypair.  Otherwise (dev or dep scriptworker), skip to the next step.
+### initial setup
```

Comment:
Not a blocker -- we can use 2 #'s instead of 3 here, since we're nesting under a single # header.
Up to you.

Reply:
Sure, let me fix this quickly and rebuilt locally to see if renders nicely.